test/monitoring/components_metrics_test.go

```diff
@@ -90,6 +90,11 @@ func testComponentsPrometheusMetrics(t *testing.T, v1api v1.API) {
 			query:         "certmanager_controller_sync_call_count",
 			platforms:     []testutil.Platform{testutil.PlatformPacket, testutil.PlatformAWS, testutil.PlatformAKS},
 		},
+		{
+			componentName: "experimental-istio-operator",
```

Comment:
Shouldn't we be testing both istio operator metrics and also istio installation?